cli/operator/keys/import.go

```diff
@@ -0,0 +1,109 @@
+package keys
+
+import (
+	"fmt"
+	"regexp"
+	"strings"
+
+	"github.com/Layr-Labs/eigensdk-go/crypto/bls"
+	"github.com/NethermindEth/eigenlayer/cli/prompter"
+	"github.com/ethereum/go-ethereum/crypto"
+	"github.com/spf13/cobra"
+)
+
+func ImportCmd(p prompter.Prompter) *cobra.Command {
+	var (
+		keyType  string
+		insecure bool
+		help     bool
+	)
+
+	cmd := cobra.Command{
+		Use:   "import --key-type <key-type> [flags] <keyname> <private-key>",
```

Comment:
So far we have been supporting creating or importing single keys. Do we have a use case for several keys? It will be a drag to run the cmd several times for X number of keys

Reply:
Most likely there's only a need to have one ecdsa and one bls by operator to do onchain stuff even if we have multiple AVS so right now there's no need to support batch import. If we get feedback later we can do it.